docs/resources/vnic.html.md

```diff
@@ -1,11 +1,3 @@
----
```

Comment:
Shouldn't file name be just vnic.md  ???

Reply:
you are absolutely right, i pushed a new commit fixing this